core/storage/feedback/gae_models.py

```diff
@@ -183,13 +184,16 @@ def get(cls, exploration_id, thread_id, message_id, strict=True):
         return super(FeedbackMessageModel, cls).get(instance_id, strict=strict)
 
     @classmethod
-    def get_messages(cls, exploration_id, thread_id):
+    def get_messages(cls, exploration_id, thread_id, is_full_thread_id=False):
```

Comment:
Oops, my bad. Looks like I chose the easy way out, which turned out to be unsuitable for this case.
Fixed this now - removed the flag and added a property to the model to get the thread id.